docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -237,7 +237,98 @@ Step 4: Confirm Network Functionality
         MyKey: 999
 
 
-Step 5: Stop the Sawtooth Network (Optional)
+.. _configure-txn-procs-docker-label:
+
+Step 5. Configure the Allowed Transaction Processors (Optional)
+---------------------------------------------------------------
+
+By default, a validator accepts transactions from any transaction processor.
+However, Sawtooth allows you to limit which transaction processors can submit
+transactions.
+
+In this step, you will configure the validator network to accept transactions
+only from the four transaction processors in the example environment:
+IntegerKey, Settings, XO, and Validator Registry. This on-chain setting is
+configured on one validator node, then Sawtooth sends the setting to the other
+nodes in the network as a transaction in a batch.
```

Comment:
I think this last sentence gives a little too much information in this context. I think something like "Transaction type restrictions are an on-chain setting, so configuration changes are applied to all validators." gets the point across without diving into the details.